lib/BackgroundJob/Tasks/ImageProcessingTask.php

```diff
@@ -216,7 +216,7 @@ private function findFaces(\CnnFaceDetection $cfd, string $dataDir, Image $image
 	 * @return ImageProcessingContext Generated context that hold all information needed later for this image
 	 */
 	private function prepareImage(string $imagePath): ImageProcessingContext {
-		$image = new \OC_Image(null, $this->context->logger->getLogger(), $this->context->config);
+		$image = new OCP_Image(null, $this->context->logger->getLogger(), $this->context->config);
```

Comment:
I guess you already saw this weird error, I guess it starts due to this change: https://travis-ci.org/matiasdelellis/facerecognition/jobs/468457344#L835
Not sure if this is false alarm, or you just need to change type in signature of resizeImage

Reply:
I hope to fix it with the last commit ...

Just in case you doubt.. The change of OC_Image is because it is private. OCP\Image must be used, but it conflicts with FaceRecognition\Db\Image.. Therefore use aliases to differentiate them... :wink: